go.mod

```
@@ -37,7 +37,7 @@ require (
 	github.com/spacemeshos/go-scale v1.1.4
 	github.com/spacemeshos/merkle-tree v0.2.1
 	github.com/spacemeshos/poet v0.6.4
-	github.com/spacemeshos/post v0.4.1
+	github.com/spacemeshos/post v0.4.2-0.20230306123131-b29abb1875e8
```

Is there a version v0.5.0 of post?

```suggestion
	github.com/spacemeshos/post v0.5.0
```

---

There was not yet. I wanted first to ensure that the systest pass and only then make a release and bump. Sorry I will leave a note in the description next time.

It's v0.5.0 now.